pkgs/tools/networking/cntlm/default.nix

```diff
@@ -11,6 +11,11 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ which ];
 
+  preConfigure = stdenv.lib.optionalString stdenv.isDarwin ''
+    sed -e 's/gcc/clang/' Makefile > Makefile.clang
```

Comment:
Note that `makeFlags=["CC=cc"];` also should override the compiler. I am not sure if this applies here.

Reply:
Is this required in the presence of? https://github.com/framon/cntlm/blob/master/configure#L48

Reply:
Will look into the `configure` script @Mic92, thanks for pointing it out :)

I tried `makeFlags=["CC=cc"];` but that didn't work. I think it's because the `configure` script is somehow unsetting the `CC` env var.